marker/config/parser.py

```diff
@@ -32,7 +32,7 @@ def common_options(fn):
                           help="Path to JSON file with additional configuration.")(fn)
         fn = click.option("--languages", type=str, default=None, help="Comma separated list of languages to use for OCR.")(fn)
         fn = click.option("--disable_multiprocessing", is_flag=True, default=False, help="Disable multiprocessing.")(fn)
-        fn = click.option('-l', is_flag=True, help="List available builders, processors and converters")(fn)
+        fn = click.option("--skip_existing", is_flag=True, default=False, help="Skip existing files.")(fn)
```

Comment:
This only works for convert.py, right?  Maybe we should put it there

Reply:
You're right! moved!